shared/cfg/guest-os/Linux/Fedora/31.cfg

```diff
@@ -0,0 +1,107 @@
+- 31:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - riscv64:
+            # Image provided by Rich Jones, for instructions read:
+            # https://avocado-vt.readthedocs.io/en/latest/Experimental.html#riscv64
+            no install, setup, unattended_install, svirt_install
+            vm_arch_name = riscv64
+            # The provided image uses "riscv" password
+            password = riscv
+            # Double the usual timeouts
+            timeout = 1200
+            login_timeout = 720
+            kill_timeout = 120
+            # Currently we have to boot via custom kernel
+            kernel = images/f31-${vm_arch_name}-kernel
+            kernel_params = "console=ttyS0 ro root=/dev/sda"
+            virtio_blk:
+                kernel_params = "console=ttyS0 ro root=/dev/vda"
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    image_name = images/f31-${vm_arch_name}
+    os_variant = fedora31
+    # default boot path is set in ../Fedora.cfg: boot_path = "images/pxeboot"
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel = images/f31-${vm_arch_name}/vmlinuz
+        initrd = images/f31-${vm_arch_name}/initrd.img
+        # Unattended-file does not require any changes
+        unattended_file = unattended/Fedora-31.ks
+        unattended_install.url:
+            # Installation works fine with mem=1024 on methods such as cdrom
+            # but fails ("No space left on device") with methods such as url.
+            mem = 2048
+            url = http://dl.fedoraproject.org/pub/fedora/releases/31/Server/${vm_arch_name}/os/
```

Comment:
wrong link instead use this `https://dl.fedoraproject.org/pub/fedora-secondary/releases/31/Server/${vm_arch_name}/os/`

Reply:
Hi,thanks a lot for your review,I have updated the commit.